### PR TITLE
Feature/142044 metrics

### PR DESCRIPTION
**Pull request by dmitriyisaev**
Metrics + zipkin

### CODE REVIEW

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

File: conf/routes [hunk 1]
```diff
@@ -22,14 +22,6 @@ DELETE   /dslinks               controllers.MainController.removeDisconnectedLin
 PUT      /uplinks               facades.websocket.WebSocketController.uplinkHandshake(url, name)
 DELETE   /uplinks/:name         facades.websocket.WebSocketController.removeUplink(name)
 
-# Metrics
-GET      /metrics/cluster       controllers.MetricController.memberEvents(role: Option[String], address: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/connections   controllers.MetricController.connectionEvents(linkName: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/sessions      controllers.MetricController.sessionEvents(linkName: Option[String], from: Option[Long], to: Option[Long])
-GET      /metrics/requests      controllers.MetricController.requestStatsByLink(from: Option[Long], to: Option[Long])
-GET      /metrics/batches       controllers.MetricController.requestStatsByMethod(from: Option[Long], to: Option[Long])
-GET      /metrics/responses     controllers.MetricController.responseStatsByLink(from: Option[Long], to: Option[Long])
```
Comment (vananyev) [inline conf/routes:31]: Are we confirmed from our PO delete "old" rest api for metrics?
Am i got right that we provide new Rest API for metric purpose? How could i see those API?

**Comment by vananyev** [comment]
@dmitriyisaev , as i got, you removed Influx dao and JDBC dao, are you sure what it will not used in the future?

File: conf/routes (hunk 1 above)
Comment (dmitriyisaev) [reply conf/routes:31]: That functionality used only for metrics/statistics collection. I discussed it with @uralian, we desided that it's not a problem to return this functionality if we'll need it, but now it's looks unnecessary.

Comment (dmitriyisaev) [reply conf/routes:31]: And yes, old api removed since we send metrics right to the graphite server

**Review by vananyev — APPROVED** [review]

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Pull request merged**